### ISSUE TITLE
Replacing istanbul with nyc

### ISSUE THREAD

**Issue by dblVs** [monounity/karma-typescript#356]
Hey I was going through security issues in my package-lock and I noticed that istanbul is already deprecated. I thing it would be nice if karma-typescript used nyc instead.

**Comment by erikbarke** [comment]
Hey @dblVs, support for the latest Istanbul API has been release in a beta version: https://www.npmjs.com/package/karma-typescript/v/5.0.0-beta.0, it would be awesome if you could try it out on your end and report back any issues!

**Comment by dblVs** [comment]
@erikbarke ah sorry i haven't checked the beta version. I'll try it out

**Comment by dwightjack** [comment]
Hi, I tried v5.0.0-beta.0 but I encountered an unexpected behavior.

In my config, I've set up coverage reports for lconv as follows:

```js
lcovonly: {
  directory: 'coverage',
  subdirectory: () => '',
  filename: 'lcov.info',
},
```

In v4.x this was saving a coverage report to `coverage/lcov.info` which is one of the known paths scanned by CodeClimate. 

After installing the beta version The file got saved to `coverage/lcov.info/lcov.info`.

Looks like istanbul is using the path generated by the configuration as a directory instead of a file path.
